packages/api/package.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "@comet-brevo-module/api",
+    "version": "1.0.0",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/vivid-planet/comet-brevo-module/",
+        "directory": "packages/api/"
+    },
+    "license": "BSD-2-Clause",
+    "main": "lib/index.js",
+    "types": "lib/index.d.ts",
+    "bin": {
+        "comet-api-generator": "bin/comet-api-generator.js"
+    },
+    "files": [
+        "bin/**/*.js",
+        "lib/*"
+    ],
+    "scripts": {
+        "clean": "rimraf lib",
+        "build": "$npm_execpath run clean && tsc -p tsconfig.build.json",
+        "dev": "tsc --watch --preserveWatchOutput -p tsconfig.build.json",
+        "lint": "run-p lint:eslint lint:tsc",
+        "lint:eslint": "eslint --max-warnings 0 src/ package.json",
+        "lint:tsc": "tsc --noEmit",
+        "list-exports": "ts-node list-exports.ts",
+        "test": "jest",
+        "test:watch": "jest --watch"
+    },
+    "dependencies": {
+        "rimraf": "^3.0.0"
```

Comment:
This should be a dev dependency

Reply:
I moved this to dev dependencies in [44f62c5](https://github.com/vivid-planet/comet-brevo-module/pull/3/commits/44f62c58dded7a8135cbc3d6b3a71e8e7ee918a2)